### DISCUSSION TITLE
Is it possible to download old versions of applications using WinGet?

### DISCUSSION THREAD

**Discussion by ryantheleach** [microsoft/winget-cli#2086] [Q&A] [answered]
Is it possible to downgrade packages from the windows store, or at the very least, download them for inspection?

**Comment by ItzLevvie** [comment]


**Reply by denelon** [reply]
You may also run `winget show <package> --versions` to get the list of versions available in the repository.

**Reply by github-account1111** [reply]
Was the answer removed?

**Comment by denelon** [comment, marked as answer]
The Microsoft Store only offers a single version of a package at a time. There are caveats for flight rings, but generally speaking there is no notion of getting an earlier version. For some packages it is possible to download them so they can technically be evaluated / installed later, but the default behavior in the Microsoft Store is to continually check for updates and provide only the latest version available.

**Reply by ryantheleach** [reply]
This is frustrating from a security research standpoint, but not wingets problem :(.

Thanks for the reply.

**Comment by mrfoxesite4482** [comment]
Try this website and see if it helps! https://store.rg-adguard.net/
